src/components/builder/values/items-expression.tsx

```diff
@@ -32,7 +36,13 @@ export const ItemsExpression: React.FC = () => {
       }
     >
       <div>
-        <JSONEdit name={NAME} data={value} rows={3} id={htmlId} />
+        <JSONEdit
+          name={NAME}
+          data={value}
+          rows={3}
+          id={htmlId}
+          validateLogic={logic => validateLogic(logic, [['', '']])}
```

Comment:
IMO this validation should not be passed at the component level, but rather at the zod schema level. Alternatively, you could pass it in at the Formik field level: https://formik.org/docs/api/field#validate

But the end result is that validation errors from field-level validation and form-level validation is the same - they block submitting the form because it is not valid, and the validation errors are normalized inside the `errors[fieldName]` data structure (where `errors`, `values` and `touched` all share the same structure w/r to expanding the `.` etc.).